SPECS/dos2unix/dos2unix.spec

```diff
@@ -0,0 +1,355 @@
+Summary:       Text file format converters
+Name:          dos2unix
+Version:       7.4.1
+Release:       4%{?dist}
+License:       BSD
+URL:           https://waterlan.home.xs4all.nl/dos2unix.html
+Source:        https://waterlan.home.xs4all.nl/dos2unix/%{name}-%{version}.tar.gz
+Vendor:        Microsoft Corporation
+Distribution:  Mariner
+
+BuildRequires: gcc
+BuildRequires: gettext
+Provides:      unix2dos = %{version}-%{release}
+Obsoletes:     unix2dos < 5.1-1
+
+%description
+Convert text files with DOS or Mac line endings to Unix line endings and 
+vice versa.
+
+%prep
+%setup -q
+
+%build
+make %{?_smp_mflags} LDFLAGS="%{build_ldflags}"
+
+%install
+make DESTDIR=$RPM_BUILD_ROOT install
+
+# We add doc files manually to %%doc
+rm -rf $RPM_BUILD_ROOT%{_docdir}
+
+%find_lang %{name} --with-man --all-name
+
+%files -f %{name}.lang
+%license COPYING.txt 
+%doc man/man1/dos2unix.htm ChangeLog.txt
+%doc NEWS.txt README.txt TODO.txt
+%{_bindir}/dos2unix
+%{_bindir}/mac2unix
+%{_bindir}/unix2dos
+%{_bindir}/unix2mac
+%{_mandir}/man1/*.1*
+
+%changelog
+* Tue Aug 25 2020 Nicolas Ontiveros <niontive@microsoft.com> - 7.4.1-4
+- Initial CBL-Mariner import from Fedora 33 (license: MIT)
```

Comment:
The `License` field mentions BSD though?

Reply:
I had the impression that this mentioned the initial license, not the license of the spec file. Makes sense though :)